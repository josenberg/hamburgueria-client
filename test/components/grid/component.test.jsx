import React from 'react';

import renderer from 'react-test-renderer';

import Grid from '../../../app/components/grid';

test('The grid should be abble to render its children without problems', () => {
  const component = renderer.create(
    <Grid rows="1fr" columns="1fr 1fr">
      <Grid.Cell row="1" column="1">
        Left Cell
      </Grid.Cell>
      <Grid.Cell row="1" column="2">
        Right Cell
      </Grid.Cell>
    </Grid>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
