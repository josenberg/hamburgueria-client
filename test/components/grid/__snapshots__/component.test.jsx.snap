// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The grid should be abble to render its children without problems 1`] = `
<div
  className="grid"
  style={
    Object {
      "colGap": "0px",
      "display": "grid",
      "gridTemplateColumns": "1fr 1fr",
      "gridTemplateRows": "1fr",
      "rowGap": "0px",
    }
  }
>
  <div
    className="grid-cell"
    style={
      Object {
        "display": "grid",
        "gridColumn": "1",
        "gridRow": "1",
      }
    }
  >
    Left Cell
  </div>
  <div
    className="grid-cell"
    style={
      Object {
        "display": "grid",
        "gridColumn": "2",
        "gridRow": "1",
      }
    }
  >
    Right Cell
  </div>
</div>
`;
